test-infra/register.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+## DEMO REGISTRATIONS #######################################
```

Comment:
Do we really need the header and footer comment in this file?

I mean: ## DEMO REGISTRATIONS and ... ## ------

Reply:
Removed header/footer comment